src/Tasks/Microsoft.NET.Build.Tasks/ConflictResolution/ILog.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
+
+namespace Microsoft.NET.Build.Tasks.ConflictResolution
+{
+    public enum LogImportance
+    {
+        Low = MessageImportance.Low,
+        Normal = MessageImportance.Normal,
+        High = MessageImportance.High
+    }
+
+
+    public interface ILog
```

Comment:
These types don't need to be public, do they?

---
In reply to: [110278890](https://github.com/dotnet/sdk/pull/1052#discussion_r110278890) [](ancestors = 110278890)